apps/src/templates/courseRollupPages/CourseRollup.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import RollupUnitEntry from './RollupUnitEntry';
+import color from '@cdo/apps/util/color';
+
+const styles = {
+  main: {
+    width: 700
+  },
+  h1: {
+    color: color.teal
+  }
+};
+
+export default class CourseRollup extends Component {
+  static propTypes = {
+    objectToRollUp: PropTypes.string,
+    units: PropTypes.array
+  };
+
+  render() {
+    return (
+      <div style={styles.main}>
+        <h1>{`Course ${this.props.objectToRollUp}`}</h1>
```

Comment:
this does not look super translatable, because the string "Course Vocabulary" might translate to something like "Vocabulario del Curso" in Spanish. I think translatability is a Q2 goal, so not required for now, but wanted to mention it since it looked like you were making some effort to extract translatable strings here.

Reply:
Ya this was just temporary because the course and unit titles actually do a lot of the work for us. I'm fixing it in the later PRs.